src/stories/common/ImageFunctional.tsx

```
@@ -51,6 +57,8 @@ function ImageObject(props) {
         marginBottom: '200px',
         backgroundColor: getBackgroundColor(),
       }}
+      // @ts-expect-error
+      // TODO: fix upstream types in react-aspect-ratio to support ref
```

Not sure if it's intentional, but react-aspect-ratio seems to [only accept a handful of props](https://github.com/roderickhsiao/react-aspect-ratio/blob/a52f99bc8923c75199c351aea3e6804cc0ba5955/src/react-latest.tsx#L4-L8) rather than all which its underlying `div` could handle - this ultimately prevents a `ref` from being passed.

Ignoring the error here for now, though I might open a PR on the react-aspect-ratio side to fix.